cmd/zpool/zpool_main.c

```diff
@@ -32,6 +32,7 @@
  * Copyright (c) 2017, Intel Corporation.
  * Copyright (c) 2019, loli10K <ezomori.nozomu@gmail.com>
  * Copyright (c) 2021, Colm Buckley <colm@tuatha.org>
+ * Copyright [2021] Hewlett Packard Enterprise Development LP
```

Comment:
Can this be changed to conform with the "standard" syntax?:
Copyright (r) 2021 Hewlett Packard Enterprise Development LP

Reply:
@mmaybee This is the format we are instructed by HPE OSRB (opens source review board) to follow while raising the PRs.

Reply:
OK, no problem. While we prefer our standardized format, we can accept other forms dictated by legal departments :-)